src/features/home/components/modules/HomeModule.tsx

```diff
@@ -122,7 +74,23 @@ const UnmemoizedModule = ({
     )
   }
 
-  return <React.Fragment></React.Fragment>
+  if (
+    (isHighlightOfferModule(item) && !enableNewExclusivityBlock) ||
+    (isExclusivityModule(item) && enableNewExclusivityBlock)
+  )
+    return <React.Fragment></React.Fragment>
+
+  /* eslint-disable @typescript-eslint/no-explicit-any */
+  const ComponentModule: any = modules[item.type]
```

Comment:
toutes les valeurs de dictionnaires sont des composants qui ont une interface commune

question (non-blocking): pourquoi avons-nous besoin de ce `any` ?

Reply:
ça pourrait etre chouette d'avoir un truc du genre

```ts
type HomeModuleProps = {
    // la liste des props qui sont communes à tout les composants
}
```

```ts
type BusinessModuleProps = HomeModuleProps & {
    // la liste des props qui sont spécifiques à ce module
}
```

```suggestion
  const ComponentModule: FunctionComponent<HomeModuleProps> = modules[item.type]
```

Reply:
Mais du coup il manquerait les composants spécifiques à certains modules non ? supposons qu'un component A a besoin d'un B props que seul A à besoin, avec le typage `FunctionComponent<HomeModuleProps>` on pourrait pas donner le B props non ?

Reply:
je ne comprends pas

on peut merger comme ça ...

... et améliorer ça au prochain mob ?